src/.eslintrc.json

```diff
@@ -12,7 +12,11 @@
 
 	"rules": {
 		"import/extensions": [ "error", "always" ],
-		"import/no-cycle": "error"
+		"import/no-cycle": "error",
+		"indent": [ "error", "tab", {
+			"MemberExpression": 1,
+			"outerIIFEBody": 0
```

Comment:
Please use `"off"`:
```suggestion
			"outerIIFEBody": "off"
```

Reply:
When I change the value to "off", grunt does not work.

```
$ grunt
Running "eslint:dev" (eslint) task
Warning: src/.eslintrc.json:
	Configuration for rule "indent" is invalid:
	Value "off" should be integer.
```